integration-test/1491-mexican-shields.py

```diff
@@ -0,0 +1,1074 @@
+                    "ref": "VER 127",
+                    "smoothness": "intermediate",
+                    "highway": "secondary",
+                },
+            ),
+        )
+
+        self.assert_has_feature(
+            z, x, y, "roads", {
+                "id": 323155576,
+                "shield_text": "127",
+                "network": "MX:VER",
+            }
+        )
+
+    def test_mx_yuc(self):
+        import dsl
+
+        z, x, y = (16, 16692, 28851)
+
+        self.generate_fixtures(
+            dsl.is_in("MX", z, x, y),
+            # https://www.openstreetmap.org/way/131913202
+            dsl.way(
+                131913202,
+                dsl.tile_diagonal(z, x, y),
+                {
+                    "source:name": "INEGI",
+                    "maxspeed": "80",
+                    "name": u"Carretera Sucil\xe1 - Calotmul",
+                    "source:maxspeed": "INEGI",
+                    "surface": "asphalt",
+                    "source": "openstreetmap.org",
+                    "oneway": "no",
+                    "ref": "YUC 33",
+                    "highway": "secondary",
+                },
+            ),
+        )
+
+        self.assert_has_feature(
+            z, x, y, "roads", {
+                "id": 131913202,
+                "shield_text": "33",
+                "network": "MX:YUC",
+            }
+        )
+
+    def test_mx_zac(self):
+        import dsl
+
+        z, x, y = (16, 14024, 28373)
+
+        self.generate_fixtures(
+            dsl.is_in("MX", z, x, y),
+            # https://www.openstreetmap.org/way/29183967
+            dsl.way(
+                29183967,
+                dsl.tile_diagonal(z, x, y),
+                {
+                    "name": "Carretera Plateros-La Salada-Rancho Grande",
+                    "surface": "asphalt",
+                    "source": "openstreetmap.org",
+                    "oneway": "no",
+                    "ref": "ZAC 77",
+                    "highway": "secondary",
+                },
+            ),
+        )
+
+        self.assert_has_feature(
+            z, x, y, "roads", {
+                "id": 29183967,
+                "shield_text": "77",
+                "network": "MX:ZAC",
+            }
+        )
+
+    def test_mx_cmx_circuito_interior(self):
+        import dsl
+
+        z, x, y = (16, 14720, 29154)
+
+        self.generate_fixtures(
+            dsl.is_in('MX', z, x, y),
+            # https://www.openstreetmap.org/way/123899735
+            dsl.way(123899735, dsl.tile_diagonal(z, x, y), {
+                'name': 'Circuito Interior', 'source': 'openstreetmap.org',
+                'alt_name': 'Circuito Bicentenario', 'oneway': 'yes',
+                'foot': 'no', 'ref': 'INT', 'highway': 'motorway',
+            }),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 123899735,
+                'shield_text': type(None),
+                'network': 'MX:CMX:INT',
```

Comment:
Are there any of these that are just `MX:CMX` to test for?

Reply:
I couldn't find any. Mexico City seems to have a bunch of different `ref`s, e.g: [3 NTE](https://www.openstreetmap.org/way/332816679), [Eje Central](https://www.openstreetmap.org/way/24706165), [Periférico](https://www.openstreetmap.org/way/182927016). Unfortunately, I wasn't able to find any more information about them on the internet, such as whether they're any kind of official designation of road network. For the moment, I've left them out - please follow up with an issue if they ought to be `MX:CMX` networks.